doc/classes/Dictionary.xml

```diff
@@ -47,7 +47,7 @@
 			<argument index="0" name="key" type="Variant">
 			</argument>
 			<description>
-				Erase a dictionary key/value pair by key. Do not erase elements while iterating over the dictionary.
+				Erase a dictionary key/value pair by key. Returns [code]false[/code] if given key is not in the dictionary, else [code]true[/code]. Do not erase elements while iterating over the dictionary.
```

Comment:
I think it would be better to word it affirmatively (as that's what is usually done in other places):

```suggestion
				Erase a dictionary key/value pair by key. Returns [code]true[/code] if the given key was present in the dictionary, [code]false[/code] otherwise. Do not erase elements while iterating over the dictionary.
```

Reply:
Done